lib/loga/service_loggers/logger_utilities.rb

```diff
@@ -0,0 +1,9 @@
+module Loga
+  module ServiceLoggers
+    module LoggerUtilities
+      def duration(start)
```

Comment:
Given this is only used in one place we can probably inline that inside the `job_logger` class